examples/vue/src/pages/ui/components/authenticator/custom-sign-up-fields/index.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import Amplify from 'aws-amplify';
+import { toRefs } from 'vue';
+import {
+  Authenticator,
+  SignUpFormFields,
+  // Vue Composable to get access to validation errors
+  useAuthenticator,
+  // Amplify UI Primitives to simplify the custom fields
+  BaseTextField,
+  BaseCheckBox,
```

Comment:
Since this is our documented public API, I think we should figure out our naming & guidance for the future as well.

https://v3.vuejs.org/style-guide/#base-component-names-strongly-recommended are great recommendations for apps, but I do wonder what a 3rd party library primitives should be?

For comparison, Vuetify uses `v-alert` – https://vuetifyjs.com/en/components/alerts/

To avoid conflicts, does this seem reasonable?

- `base-checkbox` is a *customer's* base checkbox implementation.
- `v-alert` is from Vuetify (hence the `v`)
- `amplify-checkbox` is _ours_.

Reply:
OK, I can do that. I can add `amplify` to it. I will leave the internal ones with `base` though, unless they get exposed to the customer.

Reply:
Pushed up :) @ericclemmons